index.js

```diff
@@ -51,27 +54,18 @@ var MochaParallelTests = module.exports = function (options) {
     }
 
     options._.forEach(function (testPath) {
-        var isDirectory = statSync(testPath).isDirectory();
-        var paths = [];
+        glob(testPath, function (err, files) {
```

Comment:
glob is far more better for files search and it's probably used inside mocha: https://github.com/mochajs/mocha/blob/master/package.json#L282